src/main/java/org/folio/dcb/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package org.folio.dcb.exception;
+
+public class ResourceNotFoundException extends RuntimeException {
```

Comment:
I believe, we shouldn't multiply particular exception descriptors but use common org.folio.spring.exception.NotFoundException instead.
In other words, such a class is redundant.